### ISSUE TITLE
Missing Provider Block

### ISSUE THREAD

**Issue by zehicle** [phoenixnap/terraform-provider-pnap#1]
There's no provider block to pass credentials.  As far as I can tell, this provider assumes that I've built a local credential file.

The other providers I use, have a provider block where I pass in credentials.  That's a requirement for our use case.  It's also the pattern that's encouraged by Hashicorp and implemented consistently in others.

This is what I'd expect:
```
provider "pnap" {
  clientId: <enter your client id>
  clientSecret: <enter your client secret>
}
```

**Comment by zehicle** [comment]
while this is less desirable, it is possible to work around this problem using a local_file resource block to create the configuration file dynamically.  You may want to include this in the documentation.

# Create a config file
```
resource "local_file" "credentials" {
    sensitive_content  = "clientId: [123]\nclientSecret: [shhh]"
    filename = pathexpand("~/.pnap/config.yaml")
}
```

Also, it would be smart to include Terraform v0.13+ a resource block in the examples:
```
terraform {
  required_providers {
    local = {
      source = "hashicorp/local"
      version = ">= 2.1.0"
    }
    pnap = {
      source = "phoenixnap/pnap"
      version = ">= 0.6.0"
    }
  }
}
```

**Comment by peterb-ccbill** [comment]
@zehicle we have backlogged the issue and we'll issue a fix shortly

**Comment by pajuga** [comment]
Issue resolved in version 0.0.7
